src/GameFace/Struct/UserProfile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace GameFace.Struct
+{
+    public static class Helper
+    {
+        public static List<UserRecords> AddElement(List<UserRecords> list, UserRecords element)
+        {
+            var found = false;
+            if (list.Count == 0)
```

Comment:
This nested if can be simplified by getting rid of the count == 0 check. If the list is empty, the for loop won't do anything and therefore found will never be set to true, which will cause the element to be added to the list anyway.

While the performance impact of checking the count is so small that it doesn't matter, removing the if/else makes the method a little cleaner.

Cleaner methods are a good thing because they are easier to understand and to maintain.